client/my-app/src/components/visitapartment.js

```diff
@@ -0,0 +1,149 @@
+    <input type='time'class='input-field'placeholder='Exit At' required></input>
+    <input type='number'class='input-field'placeholder='Phone' required></input>
+    <input type="number" class='input-field' placeholder='Zip Code' required></input>
+
+              <button type='submit'class='submit-btn'>Submit</button>
+     </form>
+      </div>
+  </div>
+  <center>
+    <table>
+        <tr>
+          <th>Visitor</th>
+          <th>Visiting Apartment Id</th>
+          <th>Entry At</th>
+          <th>Exit At</th>
+          <th>Approval</th>
+          <th>Actions</th>
+        </tr>
+    <tr>
+      <td>Michael Kors</td>
+      <td>12</td>
+      <td>7:30 AM</td>
+      <td>11:20 PM</td>
+      <td>Pending</td>
+      <td>
+        <button><img src="assets/icons/pencil.png" alt="edit image" width="20" height="20"></img></button>
+      <button><img src="assets/icons/trash.png" alt="Delete image" width="20" height="20"></img></button>
+    </td>
+    </tr>
+    <tr>
+      <td>Rohit padwal</td>
+      <td>13</td>
+      <td>8:30 AM</td>
+      <td>11:20 PM</td>
+      <td>Pending</td>
+      
+    
+      <td>
+        <button><img src="assets/icons/pencil.png" alt="edit image" width="20" height="20"></img></button>
+      <button><img src="assets/icons/trash.png" alt="Delete image" width="20" height="20"></img></button>
+    </td>
+    </tr>
+    <tr>
+      <td>Ernst Handel</td>
+      <td>14</td>
+      <td>05:00 PM</td>
+      <td>9:00 PM</td>
+      
+      <td>Pending</td>
+      <td>
+        <button><img src="assets/icons/pencil.png" alt="edit image" width="20" height="20"></img></button>
+      <button><img src="assets/icons/trash.png" alt="Delete image" width="20" height="20"></img></button>
+    </td>
+    </tr>
+    <tr>
+      <td>Island Trading</td>
+      <td>15</td>
+      <td>09:00 AM</td>
+      <td>4:00 PM</td>
+   <td>Pending</td>
+      <td>
+        <button><img src="assets/icons/pencil.png" alt="edit image" width="20" height="20"></img></button>
+      <button><img src="assets/icons/trash.png" alt="Delete image" width="20" height="20"></img></button>
+    </td>
+    </tr>
+    <tr>
+      <td>Sayali Deshmukh</td>
+      <td>16</td>
+      <td>7:30 AM</td>
+      <td>12:00 PM</td>
+      <td>Pending</td>
+      <td>
+        <button><img src="assets/icons/pencil.png" alt="edit image" width="20" height="20"></img></button>
+      <button><img src="assets/icons/trash.png" alt="Delete image" width="20" height="20"></img></button>
+    </td>
+    </tr>
+    <tr>
+      <td>Somnath Jadhav</td>
+      <td>17</td>
+      <td>09:00 AM</td>
+      <td>1:00 PM</td>
+      
+      <td>Pending</td>
+    <td>
+        <button><img src="assets/icons/pencil.png" alt="edit image" width="20" height="20"></img></button>
+      <button><img src="assets/icons/trash.png" alt="Delete image" width="20" height="20"></img></button>
+    </td>
+    </tr>
+  </table>
+</center>
+        </>
+    );
+}
+
+export default ServicesInformation;
```

Comment:
1. File Name should Be capitalized as it is similar to Component Name.
2. Each and every component you created is named as Services Information when it should be something else and same case with Export statement at the end of the file